docs/data/charts/axis/XAxisAnchorBaselineDefaults.js

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+
+const defaultXAxis = {
+  id: 'angle0',
+  scaleType: 'band',
+  dataKey: 'code',
+  position: 'bottom',
+  height: 45,
+};
+
+export default function XAxisAnchorBaselineDefaults() {
```

Comment:
To avoid removing them after a year because we forgot why it's here
```suggestion
/**
*This compoennt is for test sreenshot purpose only
*/
export default function XAxisAnchorBaselineDefaults() {
```

```suggestion

export default function XAxisAnchorBaselineDefaultsArgosVisualRegression() {
```

Reply:
Or the other option is to have a folder `data/visual-regression-tests` to avoid poluting the docs content with some argos testing

Reply:
> Or the other option is to have a folder `data/visual-regression-tests` to avoid poluting the docs content with some argos testing

Yeah, I prefer that one actually. I'll do that